src/Microsoft.ML.Transforms/CustomMappingCatalog.cs

```diff
@@ -4,6 +4,7 @@
 
 using System;
 using Microsoft.ML.Data;
+using Microsoft.ML.Runtime;
```

Comment:
Do we need this here?

Reply:
Otherwise you cannot reference `ComponentCatalog` in the xml doc.